src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.spec.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react/prop-types */
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BasicSliderInput } from './BasicSliderInput';
+import { TextFieldTesting } from '../../../../tests/MuiComponentsTesting/TextFieldTesting';
+import { SliderTesting } from '../../../../tests/MuiComponentsTesting/SliderTesting';
+
+const mockOnValueChanged = jest.fn();
+
+const defaultProps = {
+  label: 'Stock',
+  value: 5,
+  valueLabelDisplay: 'auto',
+  step: 1,
+  min: 0,
+  max: 10,
+  orientation: 'horizontal',
+  disabled: true,
+  handleSliderValueChange: mockOnValueChanged,
+};
+const propsInEditMode = {
+  ...defaultProps,
+  disabled: false,
+};
+const getRandomInvalidValue = () => {
+  const generateRandomIndex = Math.floor(Math.random() * 3);
+  const invalidValues = [null, undefined, NaN];
+  return invalidValues[generateRandomIndex];
```

Comment:
Not a huge fan of using a random value instead of checking the three cases :sweat_smile: 
It could lead to tests that fail sometimes with errors hard to reproduce.
Could we test the three cases ?

Reply:
Good remark, my craving for lines' economy isn't always a good idea :laughing: